spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -96,6 +96,13 @@ sealed class SpdxExpression {
      */
     abstract fun decompose(): Set<SpdxSingleLicenseExpression>
 
+    /**
+     * Return the [disjunctive normal form][1] of this expression.
+     *
+     * [1]: https://en.wikipedia.org/wiki/Disjunctive_normal_form
+     */
+    open fun disjunctiveNormalForm(): SpdxExpression = this
```

Comment:
As far as I can see it's not required by the definition, but it would be a nice improvement. I already thought about implement it as part of `normalize`, but did not want to do it now.

Reply:
sure.